files/zh-cn/web/api/response/url/index.md

```diff
@@ -1,52 +1,46 @@
 ---
-title: Response.url
+title: Response：url 属性
 slug: Web/API/Response/url
 ---
 
-{{APIRef("Fetch")}}
+{{APIRef("Fetch API")}}
 
-The **`url`** read-only property of the {{domxref("Response")}} interface contains the URL of the response. The value of the `url` property will be the final URL obtained after any redirects.
+{{domxref("Response")}} 接口的只读属性 **`url`** 包含了响应的 URL 地址。`url` 属性值为经过重定向后最终获得的 URL 地址。
 
-## Syntax
+## 值
 
-```plain
-var myURL = response.url;
-```
-
-### Value
-
-A {{domxref("USVString")}}.
+一个字符串。
 
-## Example
+## 示例
 
-In our [Fetch Response example](https://github.com/mdn/fetch-examples/tree/gh-pages/fetch-response) (see [Fetch Response live](http://mdn.github.io/fetch-examples/fetch-response/)) we create a new {{domxref("Request")}} object using the {{domxref("Request.Request","Request()")}} constructor, passing it a JPG path. We then fetch this request using {{domxref("GlobalFetch.fetch","fetch()")}}, extract a blob from the response using {{domxref("Body.blob")}}, create an object URL out of it using {{domxref("URL.createObjectURL")}}, and display this in an {{htmlelement("img")}}.
+在 [Fetch Response 示例](https://github.com/mdn/dom-examples/tree/main/fetch/fetch-response) 中（请参见 [Fetch Response 实时演示](https://mdn.github.io/dom-examples/fetch/fetch-response/)），我们使用 {{domxref("Request.Request", "Request()")}} 构造函数创建了一个新的 {{domxref("Request")}} 对象，并向其传递了 JPG 的路径。然后，我们使用 {{domxref("fetch()")}} 获取该请求，使用 {{domxref("Response.blob")}} 从响应中提取一个 blob，使用 {{domxref("URL.createObjectURL")}} 从中创建一个对象 URL，并将其显示在 {{htmlelement("img")}} 中。
 
-Note that at the top of the `fetch()` block we log the response `URL` to the console.
+请注意，在 `fetch()` 块的顶部，我们会将响应 `URL` 记录到控制台。
 
 ```js
-var myImage = document.querySelector("img");
+const myImage = document.querySelector("img");
 
-var myRequest = new Request("flowers.jpg");
+const myRequest = new Request("flowers.jpg");
 
-fetch(myRequest).then(function (response) {
-  console.log(response.url); // returns https://developer.mozilla.org/zh-CN/docs/Web/API/Response/flowers.jpg
-  response.blob().then(function (myBlob) {
-    var objectURL = URL.createObjectURL(myBlob);
+fetch(myRequest).then((response) => {
+  console.log(response.url); // 返回 https://developer.mozilla.org/en-US/docs/Web/API/Response/flowers.jpg
+  response.blob().then((myBlob) => {
+    const objectURL = URL.createObjectURL(myBlob);
     myImage.src = objectURL;
   });
 });
```

Comment:
This code example is not match with [the source file](https://github.com/mdn/dom-examples/blob/main/fetch/fetch-response/index.html). Could you also update the en-US document :)